Client/Blogger/src/Components/RegisterForm.jsx

```diff
@@ -0,0 +1,131 @@
+import { Form } from "react-bootstrap";
+import { useState } from "react";
+import countryData from "../utils/Country_data";
+import axios from "axios";
+function RegisterForm() {
+  const [firstname, setFirstName] = useState("");
+  const [lastname, setLastName] = useState("");
+  const [gender, setGender] = useState("male");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [location, setLocation] = useState("india");
+  const [phone, setPhone] = useState("");
+
+  const registerUser = async (e) => {
+    e.preventDefault();
+    const response = await axios.post("http://localhost:7000/user", {
+      firstname,
+      lastname,
+      gender,
+      email,
+      password,
+      location,
+      phone,
+    });
+    console.log(response);
+    response.data.success === false
```

Comment:
@Hareesh-Kumaran , no need to match false value with false, just write like - response.data.success ? if true : if false.